desktop/linux/index.md

```diff
@@ -24,62 +24,26 @@ To install Docker Desktop for Linux:
 1. Set up the [Docker repository](../../engine/install/ubuntu.md#install-using-the-repository).
 2. Download and install the Tech Preview Debian package:
     ```console
-    $ curl https://desktop-stage.docker.com/linux/main/amd64/74258/docker-desktop.deb --output docker-desktop.deb
+    $ curl https://desktop-stage.docker.com/linux/main/amd64/76617/docker-desktop.deb --output docker-desktop.deb
     $ sudo apt install ./docker-desktop.deb
     ```
-3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration (automated in post-install script).
 
   There are a few post-install configuration steps done through the maintainers' scripts (post-install script contained
   in the deb package.
 
   For each user, the post-install script:
 
    - installs systemd units
-   - configures `desktop-linux` as the default Docker CLI context
    - installs Compose and the `docker scan` plugins to `~/.docker/cli-plugins`
```

Comment:
This is no longer done in `postinst`, and gets installed in `/usr/lib/docker/cli-plugins`